ontologies/vocabs.ttl

```diff
@@ -1118,8 +1097,10 @@ Tag for the Entertainment industry sector"""^^xsd:string ;
 
 gist:_IndustrySector_financial-services
 	a gist:IndustrySector ;
-	rdfs:label "Financial Services"^^xsd:string ;
-	skos:altLabel "Financial-Services"^^xsd:string ;
+	skos:altLabel
+		"Financial Services"^^xsd:string ,
+		"Financial-Services"^^xsd:string
```

Comment:
Is this helpful?
Also, should there not be a stixTerm? I guess that is a different git issue?